tests/test_utils.py

```diff
@@ -255,6 +255,25 @@ def test_load_schema_org(self):
 
 
 class TestDfUtils:
```

Comment:
do we have any code that use `data_model=True` option? If so, I would recommend adding a test. If not, I would recommend removing `data_model` parameter. I saw that `_convert_csv_to_data_model` function is using `data_model=True` option, but it doesn't seem like `_convert_csv_to_data_model` is being used.

Reply:
Oh never mind. I just noticed that `parse_csv_model` function uses load_df and data_model=True. Since you are adding test for `load_df`, do you think you could also write a test to cover situations where data_model = True? If not, could you open a new ticket?

Reply:
Thanks for confirming @linglp! I've opened up a ticket to cover that parameter's use